unittests/test_ppo.py

```diff
@@ -0,0 +1,52 @@
+import unittest
```

Comment:
Now this is a useful class but I think we should be handling unit in a separate PR....

Reply:
I kept it in this merge because it tests the ModelBranch implementation

Reply:
got it. It needs a lot of work. lets chat later.